test_data/fb_batch_write.py

```diff
@@ -0,0 +1,54 @@
+import sys
+import json
+from datetime import datetime
+from firebase_admin import credentials, initialize_app
+from google.cloud import firestore
+
+
+def import_data_to_firestore(db, test_user, data):
+    collection_path = f"user/{test_user}/item"
```

Comment:
I assume that the username must already exist in the firebase for batch write to work right?

Reply:
Added a function that creates the user document if it does not exist, allowing this script to be ran for any username 💪